content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,276 @@ ifndef::include-only-once[]
+* Notes: 
+
+Label::
+(To be defined)
+
+* Source: 
+* Notes: There is no definition for "Label" in OpenLABEL.
+
+Lidar sensor::
+(To be defined)
+
+* Source:
+* Notes:
+
+Material assignment::
+A mapping of material names to parts of 3D models.
+
+* Source: 
+* Notes:
+
+Material classification::
+Sorting materials into classes.
+
+* Source: 
+* Notes:
+
+Material property::
+Intensive physical or chemical properties of a material 
+
+* Source: 
+* Notes: Open Point: What about geometric properties, i.e. roughness, thickness?
+
+Material::
+(To be defined)
+
+* Source:
+* Notes:
+
+Measurement::
+(To be defined)
+
+* Source: 
+* Notes: We would take different meanings compared to an automotive testing domain.
+
+Model::
+(To be defined)
+
+* Source: 
+* Notes: Differentiate between 3D model and other models, e.g. sensor models, logics models, vehicle models.
+
+Multispectral/hyperspectral::
+Wide range of wavelengths.
+
+* Source: 
+* Notes:
+
+Normals (face, vertex normals)::
+(To be defined)
+
+* Source: 
+* Notes: Look up normal vector on surface.
+
+Object::
+An object ID can be assigned, discrete meshes, defined 3D boundaries.
+
+* Source: 
+* Notes:
+
+Object Part / Subtype::
+(To be defined)
+
+* Source: 
+* Notes: What is an object and what is an object part?
+
+Object Type::
+Class of objects with common properties
+
+* Source: 
+* Notes:
+
+Open loop::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Opacity | Translucence | Transparency::
+(To be defined)
+
+* Source: 
+* Notes: Taken over from terminology collection list on Sharepoint.
+
+Parameter File::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Path tracing::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Perception::
+"Machine perception is to reliably detect all the other road users relevant to the operation of the automated driving, and to assign them correctly to the traffic infrastructure."
+
+* Source: Klaus Dietmayer, Predicting of Machine Perception for Automated Driving, 2016
+* Notes:
+
+Physical (as in physical parameter)::
+A physical property is any property of a physical system that is measurable. 
+
+* Source: Mark, Burgin (2016-10-27). Theory Of Knowledge: Structures And Processes
+* Notes:
+
+Post (Rendering) Effects::
+Rendering techniques to improve fidelity while maintaining performance, that are typically non-physical (or in other words fake)
+
+* Source: 
+* Notes:
+
+Radar sensor::
+(To be defined)
+
+* Source:
+* Notes:
+
+Ray tracing::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Real time / offline::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Roughness::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Scene::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Scenario.::
+A description of the behavior or temporal evolution of physical objects and environmental conditions on the driving infrastructure over an interval of time, including the movement of traffic participants or the change of environmental conditions.
+
+* Source: 
+* Notes: OpenSCENARIO standard, 2024
```

Comment:
Is this really a note or is it the source @FKlopfer ?

Reply:
Sorry, mixed this up, OpenSCENARIO is the source. To be more precise, it is OpenSCENARIO DSL 2.1.0 (and not OpenSCENARIO XML).